Chinese/DesertHell/OldStory/Story.md

```diff
@@ -1,10 +1,7 @@
 从前有座山，
 山上有座庙，
 庙里有个白胡子老头正在讲故事，
-他说,
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.